## Intro
- For an exception thrown from proceeding join point. 
    - You can handle / swallow / stop the exception. 
    - Or you can simply rethrow the exception. 
- This gives you fine grained control over how the target method is called. 